src/rules/media-feature-name-no-unknown/README.md

```diff
@@ -0,0 +1,79 @@
+# media-feature-name-no-unknown
+
+Disallow unknown media feature names.
+
+```css
+@media (min-width: 700px) { }
+/**     ↑
+ * These media feature names */
+```
+
+This rule considers media feature names defined in the CSS Specifications, up to and including Editor's Drafts, to be known.
+
+All vendor-prefixes media feature names are ignored.
+
+Caveat: Media feature names within a range context are currently ignored.
```

Comment:
I really don't understand why we would put it off. If you want to add a description to another rule we can just do that separately. It just makes some extra work to add another PR.

> On Aug 9, 2016, at 6:06 AM, Evilebot Tnawi <notifications@github.com> wrote:
> 
> In src/rules/media-feature-name-no-unknown/README.md:
> 
> > @@ -0,0 +1,79 @@
> > +# media-feature-name-no-unknown
> > +
> > +Disallow unknown media feature names.
> > +
> > +```css
> > +@media (min-width: 700px) { }
> > +/**     ↑
> > + * These media feature names */
> > +```
> > +
> > +This rule considers media feature names defined in the CSS Specifications, up to and including Editor's Drafts, to be known.
> > +
> > +All vendor-prefixes media feature names are ignored.
> > +
> > +Caveat: Media feature names within a range context are currently ignored.
> @davidtheclark i think we should this in next PR, in media-feature-name-case we don't have link or description about this.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>